File: AR-VR/Nandini-Maheshwari_NandiniMaheshwari_2125CSE1037_2/Patterns/Pattern1.cpp
#include <iostream>
using namespace std;
// xyz
int main(){
    int n;
    cin >> n;
    
   //write your code here
   
   for(int i = 1; i<=n; i++){
       for(int j = i; j>0; j--){
           cout << "*\t";
       }
       cout << "\n";
   }
   return 0;
}

File: AR-VR/Samikshagp_SamikshaGupta_2125cse1177_2/Patterns/pattern1.java
import java.util.*;

public class Main {

    public static void main(String[] args) {
        Scanner scn = new Scanner(System.in);
        int n=scn.nextInt();
        if(n>=1 && n<=100)
        {
            for(int i=1;i<=n;i++)
            {
             for(int y=1;y<=i;y++)
             {
        System.out.print("* ");
             }
             System.out.println();
            }
        }

        // write ur code here

    }
}